.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,19 +8,19 @@
 
 </details>
 
-## ⚠️ How does this PR impact the user? ⚠️
+<!--
+⚠️ How does this PR impact the user? ⚠️
 Describe (in plain English, not technical Jargon) how this improves the user experience. If you can't tie it back to a real tangible, user goal or describe it in plain english, it's a hint that this is likely not needed and is probably an "engineering nit". 
 
-### GOOD:
+✅ GOOD:
 "As a user, I need to serve models faster. This PR focuses on enabling speed gains by using GPUs"
 
-### BAD:
+⛔️ BAD:
 "This PR enables GPUs". 
 This is bad because the *user problem* is not clear... instead it just jumps to the solution without any context. 
 
 PRs without this will not be merged.
-
----- 
+-->
```

Comment:
these instructions are by design AFAIK

Reply:
but once a PR is open you do not need then, right?

Reply:
agree, let's also loop in @williamFalcon since he was the original author of this template.